release-notes/opendistro-for-elasticsearch-trace-analytics.release-notes-1.12.0.0.md

```diff
@@ -0,0 +1,12 @@
+## 2020-11-18 Version 1.12.0.0
+
+### Features
+* Rebase trace analytics to main branch ([#1](https://github.com/opendistro-for-elasticsearch/trace-analytics/pull/1))
```

Comment:
just curious, is this manually added or auto-generated

Reply:
I'm doing it manually which doesn't take much time using macros, but there's also github [api](https://api.github.com/repos/opendistro-for-elasticsearch/trace-analytics/pulls?state=closed) or [release-drafter](https://github.com/release-drafter/release-drafter)